std/stdio.d

```diff
@@ -3416,7 +3422,7 @@ unittest
             "Hello!\nHello!\nHello!\nembedded\0null\n");
 }
 
-unittest
+@system unittest
```

Comment:
Even if @system is only needed for Windows, the change here is legitimate for the purpose of this PR, which is to simply annotate what exists. Later, people can review the @systems, figure out why they are required, and take corrective action.